File: plugin/src/main/java/ru/arc/hooks/LiteBansHook.java
package ru.arc.hooks;

import litebans.api.Database;

import java.util.UUID;

public class LiteBansHook {

    public boolean isMuted(UUID uuid, String ip) {
        return Database.get().isPlayerMuted(uuid, ip);
    }

    public boolean isBanned(UUID uuid, String ip) {
        return Database.get().isPlayerBanned(uuid, ip);
    }

}

File: src/main/kotlin/ru/arc/hooks/LiteBansHook.kt
package ru.arc.hooks

import litebans.api.Database
import java.util.UUID

class LiteBansHook {
    fun isMuted(uuid: UUID, ip: String): Boolean =
        Database.get().isPlayerMuted(uuid, ip)

    fun isBanned(uuid: UUID, ip: String): Boolean =
        Database.get().isPlayerBanned(uuid, ip)
}
